pkg/aggregator/aggregator.go

```diff
@@ -209,58 +209,84 @@ func renameDefinition(s *spec.Swagger, old, new string) {
 	delete(s.Definitions, old)
 }
 
-// Copy paths and definitions from source to dest, rename definitions if needed.
+// SafeMergeSpecs is differ from MergeSpecs as it assumers dest and source does not
+// have any name conflicts. In case of a conflict, merging will fail.
+func SafeMergeSpecs(dest, source *spec.Swagger) error {
+	return mergeSpecs(dest, source, false)
```

Comment:
In fact, it's neither safe nor unsafe. In case of conflict we return an error, right? So let's just call it `MergeSpecsOrFailOnConflict`.